lib/tasks/cap_cutover.rake

```diff
@@ -102,363 +33,13 @@ namespace :cap_cutover do
-    CapProfileIdRewriter.new.rewrite_cap_profile_ids_from_feed
-  end
-
-  desc 'utility to rewrite the authorship in the pub hash from the db tables'
-  task rewrite_authorship: :environment do
-    include ActionView::Helpers::DateHelper
-    start_time = Time.zone.now
-    total_running_count = 0
-    @rebuild_authorship_logger = Logger.new(Rails.root.join('log', 'rebuild_authorship_in_hash.log'))
-    @rebuild_authorship_logger.info 'Started resync process ' + Time.zone.now.to_s
-
-    Publication.find_each do |pub|
-      total_running_count += 1
-      pub.rebuild_authorship
-      GC.start if total_running_count % 5000 == 0
-
-      if total_running_count % 1000 == 0
-        Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time, true)}"
-        @rebuild_authorship_logger.info total_running_count.to_s + ' in ' + distance_of_time_in_words_to_now(start_time, true)
-      end
-    end
-    @rebuild_authorship_logger.info 'Finished build.' + Time.zone.now.to_s
-    @rebuild_authorship_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
-
-  desc 'utility to extract data extract data from hash to publication object, modify as needed before running'
-  task update_pubs: :environment do
-    include ActionView::Helpers::DateHelper
-    start_time = Time.zone.now
-    total_running_count = 0
-    @cap_import_logger = Logger.new(Rails.root.join('log', 'cap_update_pubs.log'))
-    @cap_import_logger.info 'Started cap update pub process ' + Time.zone.now.to_s
-
-    Publication.find_each do |pub|
-      begin
-        total_running_count += 1
-        pub_hash = pub.pub_hash
-
-        if  !pub_hash[:author].nil? && pub_hash[:author].length > 5
-          Publication.update_formatted_citations(pub_hash)
-          pub.pub_hash = pub_hash
-          pub.save
-          @cap_import_logger.info "Fixed authors for #{pub.id}"
-        end
-        GC.start if total_running_count % 5000 == 0
-
-        if total_running_count % 1000 == 0
-          Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time, true)}"
-          # @cap_import_logger.info total_running_count.to_s + " in " + distance_of_time_in_words_to_now(start_time, true)
-        end
-       rescue => e
-         @cap_import_logger.error "Problem with pub #{pub.id}"
-         @cap_import_logger.error e.inspect << "\n" << e.backtrace.join("\n")
-      end
-    end
-    @cap_import_logger.info 'Finished update.' + Time.zone.now.to_s
-    @cap_import_logger.info total_running_count.to_s + 'records were updated in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
-
-  def find_issn_id_identifiers(identifiers_array)
-    issn = nil
-    identifiers_array.each do |single_identifier_hash|
-      if single_identifier_hash[:type] == 'issn'
-        issn = single_identifier_hash[:id]
-        break
-      end
-    end
-    issn
-  end
```

Comment:
Why is `find_issn_id_identifiers` removed?  It's not part of the analysis in #356.  It's seems that it's not called anywhere else in the rake tasks.